hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/Text.java

```diff
@@ -226,6 +226,12 @@ public void set(String string) {
    * Set to a utf8 byte array.
    */
   public void set(byte[] utf8) {
+    if(utf8.length == 0){
+      bytes = EMPTY_BYTES;
+      length = 0;
+      textLength = -1;
+      return;
```

Comment:
I think I'd prefer to cut the return here and just have an `else` clause for the normal set() operation.

This isn't so long and complex a piece of code that an early return makes sense; it can be easy to miss that it's there when reading the code.

Reply:
Thank you for your advice. I'll fix it